examples/platforms/cc2538/openthread-core-cc2538-config.h

```diff
@@ -93,4 +93,102 @@
  */
 #define SETTINGS_CONFIG_PAGE_SIZE                              2048
 
+/**
+ * @def Enable support for the CC2592 range-extender front-end.
```

Comment:
The `@def` command should be followed by the macro that is being documented.

For example:
```c
/**
 * @def OPENTHREAD_CONFIG_CC2538_WITH_CC2592
 *
 * Enable support for the CC2592 range-extender front-end.
 *
 * This is a feature of the CC2538-CC2592 EM and other peripherals which
 * extends the range of the bare CC2538 to over a kilometre line-of-sight.
 * The CC2592 needs to be wired up to the RF port on the CC2538 in accordance
 * with application note 130 ("Using CC2592 Front End With CC2538", TI doc
 * SWRA447).
 *
 */
```

I see that some of the existing doxygen above makes the same mistake.

Reply:
Yeah, monkey-see-monkey-do in my case… I shall fix it.

Reply:
Yes, there are a number of similar mistakes across the `examples/`. However, `include/` and `src/` seem generally consistent (see [src/core/openthread-core-default-config.h](https://github.com/openthread/openthread/blob/master/src/core/openthread-core-default-config.h)).

As an example:
```c
/**
 * @def OPENTHREAD_CONFIG_STACK_VENDOR_OUI
 *
 * The Organizationally Unique Identifier for the Thread stack.
 *
 */
#ifndef OPENTHREAD_CONFIG_STACK_VENDOR_OUI
#define OPENTHREAD_CONFIG_STACK_VENDOR_OUI 0x18b430
#endif
```

Reply:
Ahh, gotcha … should be fixed now in 50d69ac8.